Automattic/wp-calypso
# Fix SVG props on icon components

Related to #96228

## Proposed Changes

* It fixes the SVG props on icons.

## Why are these changes being made?


* Because the props in React should be camel case and it was triggering warnings in the console.

## Testing Instructions



* Activate the feature flag: `automated-migration/pending-status`.
* Click on "Do it for me" while navigating through a migration flow.
* Open the browser console.
* Navigate to `/sites`.
* Click on the site with a pending migration.
* Make sure you don't see errors related to the icons anymore.
* And see that the icons continue working properly.

<img width="504" alt="Screenshot 2024-11-11 at 18 34 36" src="https://github.com/user-attachments/assets/20053bf7-254e-4b6e-89e9-5f6b14805312">

## Pre-merge Checklist
